.gitignore

```diff
@@ -1,4 +1,5 @@
 config.inc.php
+cache
```

Comment:
There's already a .gitignore in cache/ which should take care of this -- unless there are some file types missing from that? IIRC having .gitignore in cache/ will cause cache to be created empty on checkout, and this will cause it not to be created at all.